File: fixtures/ast-shadow/typescript/basic.ts
import { readFile } from "node:fs/promises";

export function compute(value: number): number {
  if (value === 0) {
    return 0;
  }

  for (let item = 0; item < value; item += 1) {
    while (item > 1) {
      break;
    }
  }

  switch (value) {
    case 1:
      return 1;
    default:
      return value;
  }
}

File: fixtures/ast-shadow/python/basic.py
from pathlib import Path


def compute(value: int) -> int:
    if value == 0:
        return 0

    for item in range(value):
        while item > 1:
            break

    match value:
        case 1:
            return 1
        case _:
            return value


def load_path(path: Path) -> str:
    return path.read_text()
